screens/TabTwoScreen.tsx

```diff
@@ -67,68 +94,168 @@ const App = () =>{
+          color="#841584"
+          onPress={() => getApi(URLisland)}
+        >
+          json
+        </Button>
+        <Button
+          title="Tsun Mun"
+          color="#841584"
+          onPress={() => getApi(URLtuenmun)}
+        >
+          json
+        </Button>
+        <Button
+          title="Yuen Long"
+          color="#841584"
+          onPress={() => getApi(URLYuenLong)}
+        >
+          json
+        </Button>
+        <Button
+          title="Northern"
+          color="#841584"
+          onPress={() => getApi(URLnorthern)}
+        >
+          json
+        </Button>
+        <Button title="Tai Po" color="#841584" onPress={() => getApi(URLtaipo)}>
+          json
+        </Button>
+        <Button
+          title="Sha Tin"
+          color="#841584"
+          onPress={() => getApi(URLshatin)}
+        >
+          json
+        </Button>
+        <Button title="Tsuen Wan" color="#841584" onPress={() => getApi(URLtw)}>
+          json
+        </Button>
+        <Button
+          title="Kwai Tsing"
+          color="#841584"
+          onPress={() => getApi(URLkwaitsing)}
+        >
+          json
+        </Button>
+        <Button
+          title="Sai Kung"
+          color="#841584"
+          onPress={() => getApi(URLsaikung)}
+        >
+          json
+        </Button>
+        <Button title="Kwun Tong" color="#841584" onPress={() => getApi(URLkt)}>
+          json
+        </Button>
+        <Button
+          title="Yau Tsim Mong"
+          color="#841584"
+          onPress={() => getApi(URLyautsummong)}
+        >
+          json
+        </Button>
+        <Button
+          title="Sham Shui Po"
+          color="#841584"
+          onPress={() => getApi(URLssp)}
+        >
+          json
+        </Button>
+        <Button
+          title="Kowloon City"
+          color="#841584"
+          onPress={() => getApi(URLklc)}
+        >
+          json
+        </Button>
+      </View>
       <TextInput
         onChangeText={(item) => searchFilter(item)}
         style={styles.searchInput}
-        placeholder="Search Here" />
-      <FlatList 
-        data={filterItems} 
-        style = {styles.flatListStyle}
-        renderItem={( {item} ) => (
-                <>
-                  <Text>學校中文名稱: {item["name-zh"]}</Text>
-                  <Text>學校英文名稱: {item["name-en"]}</Text>
-                  <Text>地址: {item["address-zh"]} {item["address-en"]}</Text>
-                  <Text>資助種類: {item["finance-type-zh"]} {item["finance-type-en"]}</Text>
-                  <Text>就讀學生性別: {item["student-gender-zh"]} {item["student-gender-en"]}</Text>
-                  <Text>聯絡電話: {item["telephone"]}</Text>
-                </>
-        )
-      }
-      ItemSeparatorComponent={renderSeparator}
+        placeholder="Search Here"
       />
-    </View> 
-    ); 
-  }
+      <FlatList
+        data={filterItems}
+        style={styles.flatListStyle}
+        renderItem={({ item }) => (
```

Comment:
Nit: Please add typing on `item`  to allow compile time property access with the `dot` syntax.